test/testrunner.py

```diff
@@ -177,7 +193,10 @@ def main():
             # Note:- Running multiple pytest_cmds in a sequence will result in the execution log having two
             #        separate pytest reports, both of which must be examined in case of a manual review of results.
             cmd_exit_statuses = [pytest.main(pytest_cmd) for pytest_cmd in pytest_cmds]
-            sys.exit(0) if all([status == 0 for status in cmd_exit_statuses]) else sys.exit(1)
+            if all([status == 0 for status in cmd_exit_statuses]):
+                sys.exit(0)
+            else:
+                raise RuntimeError(pytest_cmds)
```

Comment:
nit: we can probably define a better error. Perhaps DLCPyTestFailure, and maybe include some more context about what we are raising, i.e.

```
raise DLCPyTestFailure(f"Encountered a nonzero exit status when executing pytest.main with the following commands: {pytest_cmds}")
```

Reply:
I'm also OK with just logging the cmds if we are exiting 1 (i.e. LOGGER.error) -- thoughts @nskool ?

Reply:
I'm fine with the RuntimeError, since we're explicitly raising it as one exception.